README.md

```diff
@@ -11,20 +11,22 @@ retrieved from an [`issue_comment` webhook event][issue_comment_wh]) for certain
 and rerunning jobs matching those commands.
 
 Some notes:
+
 - The PR either must have an `ok-to-test` label present on the PR, or the user who writes a command must
 be an organization member, or repo owner, contributor, or collaborator.
   - Typically `ok-to-test` can/should only be applied by repo reviewers/approvers to prevent spam and abuse.
 - `rerun-actions` should only be run on comment creation. See the below [examples](#examples) for how to do this.
 
 ## Comment commands
 
-These commands are named after
+The following commands are supported by this action:
+
+- `/rerun-all` - rerun all failed workflows.
+- `/retest-workflow <workflow name>` - rerun a specific failed workflow.
```

Comment:
```suggestion
- `/rerun-workflow <workflow name>` - rerun a specific failed workflow.
```

Reply:
thanks - I don't how I missed that locally lol